GenderPayGap.WebUI/Controllers/ViewingController.cs

```diff
@@ -323,14 +323,31 @@ public IActionResult Employer(string employerIdentifier)
             ReportBackUrl = null;
 
             ViewBag.BasketViewModel = new CompareBasketViewModel {CanAddEmployers = true, CanViewCompare = true};
+            
+            int totalEntries = organisationLoadingOutcome.Result.GetRecentReports(Global.ShowReportYearCount).Count() + 1; // Years we report for + the year they joined
+            int maxEntriesPerPage = 10;
+            int totalPages = (int)MathF.Ceiling(totalEntries / maxEntriesPerPage);

```

Comment:
I think you've got some Unity showing through 😄
I'd never heard of `MathF` before, looks like it's a Unity library that's a copy of `Math` except it uses floats instead of doubles. Normally we'd just use `Math`.